fe/fe-core/src/main/java/org/apache/doris/nereids/StatementContext.java

```diff
@@ -51,6 +51,8 @@ public class StatementContext {
 
     private final IdGenerator<ObjectId> objectIdGenerator = ObjectId.createGenerator();
 
+    private final IdGenerator<ObjectId> cteIdGenerator = ObjectId.createGenerator();
```

Comment:
u need create a new class named CteId, and use CteId directly, not use primitive type int